.github/workflows/person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+
+  attr_reader :id, :name, :age
+  attr_writer :name, :age
+
+  def initialize (name="unknown",age,parent_permission=true)
+    @id=rand(1..1_000)
+    @name=name
+    @age=age
+    @parent_permission=parent_permission
+  end
+
+  def can_use_service?
+    is_of_age? || @parent_permission
+  end
+  private
+
+  def is_of_age?
+    @age>=18
+  end
+
```

Comment:
- [ ] Let's try to be a bit more organized. Kindly remove this class from the `.github/workflows` as that folder is designated for setting up linters and not for actual code.